src/player.ts

```diff
@@ -507,4 +515,18 @@ export class Player extends NativeInstance<PlayerConfig> {
     }
     return PlayerModule.canPlayAtPlaybackSpeed(this.nativeId, playbackSpeed);
   };
+
+  initDecoderConfig = () => {
```

Comment:
This is most def not the public Player API.
```suggestion
  private initDecoderConfig = () => {
```

I would probably also hold this function close to the initialize function.